test/relayserver/RelayServer.test.ts

```diff
@@ -27,7 +27,7 @@ const { expect, assert } = chai.use(chaiAsPromised).use(sinonChai)
 const TestVerifierConfigurableMisbehavior = artifacts.require('TestVerifierConfigurableMisbehavior')
 const TestDeployVerifierConfigurableMisbehavior = artifacts.require('TestDeployVerifierConfigurableMisbehavior')
 
-const revertReasonSupported = false
+const revertReasonSupported = true
```

Comment:
rather than having this `const` as `true`, wouldn't it be better to remove it and have the affected test like this?:

```typescript
    describe('#validateViewCallSucceeds()', function () {
      // RelayHub contract
      it('should fail to relay rejected transaction', async function () {
        const req = await env.createRelayHttpRequest()

        req.metadata.signature = INCORRECT_ECDSA_SIGNATURE
        const method = env.relayHub.contract.methods.relayCall(req.relayRequest, req.metadata.signature)

        try {
          await env.relayServer.validateViewCallSucceeds(method, req, 2000000)
          assert.fail()
        } catch (e) {
          assert.include(e.message, 'signature mismatch')
        }
      })
    })
```